### PR TITLE
Brand changes

### PR DESCRIPTION
**Pull request by akisblack**

### CODE REVIEW

File: src/routes/timeline.svelte [hunk 1]
```diff
@@ -150,4 +150,20 @@
...
+		<TimelineSeparator>
+			<TimelineDot
+				style={'background-color: #fff; '}
+			>
+			</TimelineDot>
+			<TimelineConnector />
+		</TimelineSeparator>
+		<TimelineContent>
+			<h3> Project Segfault decides to expand and now develops software. </h3>
+			<p> This started after Odyssey decided to work on segfautils. An API for Project Segfault. Midou followed suit with another rather big project that is (not) being developped right now. (hint: this will help people with many servers or mini servers 👀)  </p>
```
Comment (Odyssey346) [inline src/routes/timeline.svelte:166]: remove the whitespaces, it's Segfautils, also maybe "Midou is going to make a huge project in the near future together with the team that can help people with multiple servers"?

File: src/routes/timeline.svelte [hunk 2]
```diff
@@ -54,8 +54,8 @@
             <TimelineConnector />
 		</TimelineSeparator>
 		<TimelineContent>
-			<h3>Midou meets a guy named MrLeRien</h3>
-			<p>This eventually formed a nice little (primarily focused on hosting) project called mutahar.rocks. Here, the duo focused on hosting anything they could on a Core 2 Duo HP machine with 8GB of RAM that was running Arch Linux. Eventually, they decided to buy a couple of VPSes.</p>
+			<h3>MrLeRien opens a small homeserver with the help of Midou</h3>
+			<p>This eventually formed a nice little (primarily focused on hosting) project called mutahar.rocks. Here, the duo focused on hosting anything they could on a Core 2 Duo HP machine with 8GB of RAM running Arch Linux. This alongside a small VPS we're already had.</p>
```
Comment (Odyssey346) [inline src/routes/timeline.svelte:58]: "we're"? No, it's "we".

Comment (Odyssey346) [reply src/routes/timeline.svelte:58]: or, since the thing is kind of in the past, maybe "they"?

**Review by Midou36O — APPROVED** [review]

**Review by Odyssey346 — APPROVED** [review]
good maybe

**Pull request merged**